client/src/components/Sidebar.tsx

```diff
@@ -286,8 +287,17 @@ const Sidebar = ({
 
           <div className="space-y-2">
             <Button className="w-full" onClick={onConnect}>
-              <Play className="w-4 h-4 mr-2" />
-              Connect
+              {connectionStatus === "connected" ? (
+                <>
+                  <RotateCcw className="w-4 h-4 mr-2" />
+                  Restart
```

Comment:
Maybe call this `Reconnect` ? So `Connect` and `Reconnect`

Reply:
Agreed, Reconnect is a better term since you will be connecting to the same process as opposed to starting a new one (previous unintended behavior).

Reply:
On the contrary! With the STDIO transport, it's starting a new process, and that's per design. I think in STDIO mode, "Restart" is the correct term. With SSE transport, I'm happy to make it "Reconnect".

Reply:
Sounds good. STDIO ~ Restart / SSE ~ Reconnect.

Reply:
implemented in https://github.com/modelcontextprotocol/inspector/pull/216/commits/da0c855ef5eb88996adb09d25b244a1824c12f9c